src/zenml/enums.py

```diff
@@ -291,3 +291,25 @@ class ModelStages(StrEnum):
     STAGING = "staging"
     PRODUCTION = "production"
     ARCHIVED = "archived"
+
+
+class ColorVariants(Enum):
```

Comment:
Also same question, any reason we aren't using String enums here?

Reply:
Save space - the only reason. 4 bytes vs 5+ (for non-unicode db, otherwise 10+)

Reply:
Done